client/src/modules/list/components/Items/InputBar/InputBar.jsx

```diff
@@ -151,6 +143,12 @@ class InputBar extends Component {
     if (isInputEmpty) {
       this.hideForm();
     }
+
+    document.removeEventListener('keydown', this.handleEscapePress);
```

Comment:
This handleEscapePress is added correctly now - only when the input gets focus, but removing is not working yet - when you close input via escape press - the listener isn't removed and it is still invoked, so I think we it should be removed onEscapePress also